Mathlib/GroupTheory/GroupAction/Quotient.lean

```diff
@@ -350,6 +350,14 @@ instance isPretransitive_quotient (G) [Group G] (H : Subgroup G) : IsPretransiti
 
 end MulAction
 
+theorem ConjClasses.card_carrier [Group G] [Fintype G] (g : G) [Fintype (ConjClasses.mk g).carrier]
+    [Fintype <| MulAction.stabilizer (ConjAct G) g] : Fintype.card (ConjClasses.mk g).carrier =
+      Fintype.card (ConjAct G) / Fintype.card (MulAction.stabilizer (ConjAct G) g) := by
```

Comment:
I think `card G` makes more sense than `card (ConjAct G)`. And perhaps it would be nicer to state this result in terms of `Finite` and `Nat.card`, to avoid all the fintype assumptions?